### ISSUE TITLE
Luminosity contrast ratio of '--rz-secondary' text with its background does not meet the minimum contrast ratio of 4.5:1 in Windows.

### ISSUE THREAD

**Issue by msftedad** [radzenhq/radzen-blazor#2200]
**Test environment:**
OS: Windows 11 Enterprise Insider Preview Version 24H2(OS Build 26120.2415)
Browser: Microsoft Edge Version 132.0.2957.26
URL: https://blazor.radzen.com/colors?theme=material)
Tool: Accessibility insights for Windows.

**User Impact:**
The low contrast ratio between the '--rz-secondary' text and its background in Windows can greatly impact users with visual impairments. This issue makes it challenging for them to read the text, which can result in missed information and a diminished user experience.

**Observation:**
This issue is also observed with Accessibility for web tool.

**Steps to Reproduce:**  

1. Open URL: https://blazor.radzen.com/colors?theme=material in the latest edge browser.
2. Navigate to '--rz-secondary' content in theme colors section.
3. Turn on the accessibility insights for windows tool.
4. Select Foreground color (#FFFFFF) and Background color (#E91E63) of 'Search' and 'more options' icon using color picker.
5. Verify whether contrast ratio between '--rz-secondary' text with its background does meet the minimum contrast ratio or not.

**Actual Result:**
Contrast ratio between '--rz-secondary' text with its background is 4.347: 1 which is less than the required ratio in Windows.

**Refer Attachment:** 
1.Contrast ratio between '--rz-secondary' text with its background does not meet the minimum contrast ratio in Windows..mp4
2.Contrast ratio between '--rz-secondary' text with its background does not meet the minimum contrast ratio in Windows..png

<img width="960" alt="Image" src="https://github.com/user-attachments/assets/bdbd98e8-c728-4487-b8f4-74657a65879f" />
https://github.com/user-attachments/assets/adb25616-7983-446a-a63c-a4aafa9be7d9

**Expected Result:**
Contrast ratio between '--rz-secondary' text with its background should be greater than or equal to the required ratio of 4.5:1 in Windows.

**Comment by yordanov** [comment]
The background color `#E91E63` has already been replaced with `#E31C65` (contrast ration of 4.54:1). The fix is in [this commit](https://github.com/radzenhq/radzen-blazor/commit/e7d8b7454f4b3c20a5d5a796571fa1b9276c7c02) and has been available since [v5.9.0](https://github.com/radzenhq/radzen-blazor/releases/tag/v5.9.0).

@msftedad update to a more recent version or clear your browser's cache.
